stdlib/public/SwiftShims/CoreFoundationShims.h

```diff
@@ -55,44 +55,6 @@ typedef unsigned char _swift_shims_Boolean;
 typedef __swift_uint16_t _swift_shims_UniChar;
 typedef __swift_uint8_t _swift_shims_UInt8;
 
-// Buffer is nullable in case the string is zero-length.
-SWIFT_RUNTIME_STDLIB_API
```

Comment:
If something is somehow depending on these directly, we can reimplement them using dlsym, or the same tricks but in a .mm file, but I think we should see if we can get away with removing them.